test/controllers/pledge_controller_test.rb

```diff
@@ -0,0 +1,89 @@
+require 'test_helper'
+
+class PledgeControllerTest < ActionController::TestCase
+  
+  before do
+    @user = create :user
+    sign_in @user
+    @patient = create :patient
+    @pregnancy = create :pregnancy, patient: @patient
+  end
+  
+  describe 'create method' do
+    before do
+      @pledge = create :pledge,
+                pledge_type: "Sample Pledge", 
+                amount: 5, 
+                other_pledge_identifier: "none", 
+                sent: DateTime.current, 
+                sent_by: @user, 
+                paid: False,
```

Comment:
I see you come from python! An annoying ruby-ism is that `true` and `false` are both lowercase -- probably change this line to `paid: false` or it'll look for an object called `False` and eat mud.